setup.py

```diff
@@ -68,6 +68,7 @@
     long_description_content_type='text/markdown',
     version=version,
     packages=find_packages(exclude=['test', 'scripts', 'examples']),
+    package_data={"autosklearn": ["py.typed"]},
```

Comment:
Is there a reason we have to give this in both `setup.py` and `MANIFEST.in`?

Reply:
I don't think so, the `package_data` method is what I tried first but it didn't work, I think the `MANIFEST.in` has precedence. I can remove it from `setup.py` and try it out.